docs/helpers/Ticks.md

```diff
@@ -23,10 +23,10 @@ If there is only 1 axis in the graph, the tick intervals are calculated automati
 
 #### Required
 
-| Property Name | Expected | Description                                                                                                                                       |
-| ------------- | -------- | ------------------------------------------------------------------------------------------------------------------------------------------------- |
-| values        | array    | Ticks values in the axis. Each tick will have an associated grid line. If they are not provided, optional tick values need to exist.              |
-| format        | string   | [Formatter](https://github.com/d3/d3-time-format/blob/master/README.md#locales) for tick values. `Note:` This is mandatory if values are provided |
+| Property Name | Expected | Description                                                                                                                                                                                                                                              |
+| ------------- | -------- | -------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
+| values        | array    | Ticks values in the axis. Each tick will have an associated grid line. If they are not provided, optional tick values need to exist.                                                                                                                     |
+| format        | string   | [Formatter](https://github.com/d3/d3-time-format/blob/master/README.md#locales) for tick values. `Note:` This is mandatory if values are provided. To suppress tick values's trailing zeros, specify '~' just before the format type. `Eg: format: "~r"` |
```

Comment:
Why do we only this specific `~` operator?

Reply:
According to d3-format documentation [here](https://github.com/d3/d3-format/blob/master/README.md#locale_format), they have specified that '~' character will trim the insignificant trailing zeros.
The below is what's defined in the documentation:
"""" The `~` option trims insignificant trailing zeros across all format types. This is most commonly used in conjunction with types r, e, s and %. For example:

d3.format("s")(1500);  // "1.50000k"
d3.format("~s")(1500); // "1.5k"            """"""

Reply:
Thanks for the clarification and d3-format documentation